woocommerce/wc-admin
# Use ReportTable in Orders report

In #928 we merged all tables that were using the common store into `ReportTable`. Some report tables were excluded from that because they load the data from other endpoints.

In this PR I'm making the _Orders report_ to use the `ReportTable` component as well, that will make it easier to implement the _Stock report_ (#174).

The only report not using `ReportTable` yet will be the _Revenue report_.

`ReportTable` has been modified to allow directly passing the `tableData` so it will not fetch it from the `/reports/` endpoint.

### Screenshots
![image](https://user-images.githubusercontent.com/3616980/49314326-ba7cdf00-f4af-11e8-8705-172b07a69583.png)

### Detailed test instructions:
- Go to the _Orders report_.
- Verify everything related to the table is still working fine (pagination, sorting, etc.).
